developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -8,62 +8,34 @@ image: og/docs/client-libraries.jpg
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
-import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
 
 
 :::note Typescript client version
 The current Typescript client version is `v||site.typescript_client_version||`
 :::
 
-The `v3` Weaviate typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level.
+The `v3` Weaviate Typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level. 
+To start, `v3` refers to the client with the package name `weaviate-client` and `v2` refers to the package name `weaviate-ts-client`. 
```

Comment:
Consider:
The package names are different. The `v3` client package is `weaviate-client`. The `v2` client is `weaviate-ts-client`.

Reply:
When the other v3 client packages are available, we can expand this section to add them.